package-scripts.js

```diff
@@ -47,12 +47,9 @@ module.exports = {
     // default is run when you run `nps` or `npm start`
     default: 'next start',
     dev: 'next',
-    build: Object.assign(
-      {
-        default: concurrent.nps(...supportedLocales.map(s => `build.${s}`)),
-      },
-      localeBuilds
-    ),
+    build: Object.assign(localeBuilds, {
```

Comment:
This busted things when you run: `npm start build.de` which would assume you were prefixing for `default`, which we're not, we want to build the german version.